__tests__/endpoints/resourcePut.test.js

```diff
@@ -51,12 +51,12 @@ describe('PUT /resource/:resourceId', () => {
     expect(res.body).toEqual(resBody)
     const saveResource = {...resource}
     delete saveResource._id
-    saveResource.timestamp = new Date()
+    saveResource.timestamp = new Date().toISOString()
```

Comment:
Why is this changing?

Reply:
Because without the toISOString, it isn't quoted in the returned json and the tests won't pass. The other side of this is we can't put an unquoted string in the JSON as it won't open (invalid) in the test. If you have ideas to get around this, that would be helpful.